application/cmdbabstract.class.inc.php

```diff
@@ -176,6 +176,8 @@ abstract class cmdbAbstractObject extends CMDBObject implements iDisplay
 	 */
 	protected $sDisplayMode;
 	protected $aFieldsMap;
+    /*store posted values in order to be used in GetAttributeFlag*/
```

Comment:
Please use proper phpdoc:
```suggestion
    /**
     * @var array Store posted values in order to be used in GetAttributeFlag
     */
```